build/android/generate_xwalk_core_library.py

```diff
@@ -81,12 +81,9 @@ def CopyJSBindingFiles(project_source, out_dir):
   jsfiles_to_copy = [
       'xwalk/experimental/launch_screen/launch_screen_api.js',
       'xwalk/experimental/presentation/presentation_api.js',
-      'xwalk/runtime/android/core_internal/src/org/xwalk/core/'
-      + 'internal/extension/api/contacts/contacts_api.js',
-      'xwalk/runtime/android/core_internal/src/org/xwalk/core/'
-      + 'internal/extension/api/device_capabilities/device_capabilities_api.js',
-      'xwalk/runtime/android/core_internal/src/org/xwalk/core/'
-      + 'internal/extension/api/messaging/messaging_api.js'
+      'xwalk/runtime/android/core_internal/extension/api/contacts/contacts_api.js',
+      'xwalk/runtime/android/core_internal/extension/api/device_capabilities/device_capabilities_api.js',
+      'xwalk/runtime/android/core_internal/extension/api/messaging/messaging_api.js'
```

Comment:
And how about launch_screen and presentation apis? I think all those js files should be under protection of condition that `disable_android_buildin_extensions`